### ISSUE TITLE
Support for constant values

### ISSUE THREAD

**Issue by nicojs** [bcherny/json-schema-to-typescript#263]
Support const values (new in v6). https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values

Input

```json
{
  "$id": "https://api.nn-group.com/t/nn-nl/2/json_schemas/forwardcache_notify_participation",
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "Foo",
  "required": ["bar"],
  "additionalProperties": false,
  "properties": {
    "bar": {
      "const": true
    }
  }
}
```

Expected output:

```ts
export interface Foo {
  bar: true;
}
```

Current output:

```ts
export interface Foo {
  bar: {
    [k: string]: any;
  };
}
```

This feature was mentioned in #120 but as the scope of that is a bit too big for beginning contributors I thought I would split this feature out.

Any hints for json-schema-to-typescript newbies would be much appreciated.

**Comment by djbeaumont** [comment]
I've been hoping someone would give this a try for about a year now and I've finally had a chance to give this a go in the PR above https://github.com/bcherny/json-schema-to-typescript/pull/264

**Comment by bcherny** [comment]
Fixed in #290, published in v10.1.0.
